main.js

```diff
@@ -270,22 +304,27 @@ const fetchHandler = async function (req) {
 };
 
 // Start a Bun web server with fetchHandler() as the function to handle requests
-const server = Bun.serve({
-  port: 8080,
+var servercfg = {
```

Comment:
This should probably be a `const`.

Reply:
Are you sure? It's setting servercfg.tls shortly after, that doesn't work with it being a const no?

Reply:
It should be a const
Because for a constant you cant change **what** its referencing to, but you can change the referenced objects properties for example, because the object that its referring to is still the same object

Reply:
In short it should be a constant

Reply:
uhhh okay